exporter/opentelemetry-exporter-jaeger/tests/test_jaeger_exporter_thrift.py

```diff
@@ -152,9 +175,17 @@ def test_constructor_by_environment_variables(self):
         exporter.password = None
         self.assertNotEqual(exporter._collector_http_client, collector)
         self.assertTrue(exporter._collector_http_client.auth is None)
-
         environ_patcher.stop()
 
+    @patch("opentelemetry.exporter.jaeger.trace._TRACER_PROVIDER", None)
+    def test_constructor_with_no_traceprovider_resource(self):
+
+        """Test the constructor when there is no resource attached to trace_provider"""
```

Comment:
added this new test to check scenario when tracerprovider has no resource attribute.